client/docs/pom.xml

```diff
@@ -33,7 +33,7 @@
               </goals>
               <configuration>
                 <files>
-                  <file>${project.basedir}/../.github/project.yml</file>
+                  <file>${project.basedir}/../../.github/project.yml</file>
```

Comment:
Where is the documentation for the server part? We might need to review the README to add links to both. The client side has been migrated to quarkiverse: https://docs.quarkiverse.io/quarkus-openapi-generator/dev/index.html

We might either:

1. Review this doc to include "Quarkus OpenAPI Generator - Client"
2. Have a unique docs dir with both docs

If you already have docs, we can link both (client and server) in the readme, but still 1 should be done here. wdyt?

Reply:
We don't have docs yet (I'm working on having something decent, but things keep happening :(). I have updated the reference in the doc you mentioned. If you're happy with the structure proposed, maybe it's better to merge this PR and then add the docs in a follow-up one? wdyt?